sidecar/main.cpp

```diff
@@ -45,11 +45,6 @@ int GetSettings(nlohmann::json& json) {
           break;
         }
       }
-
-      if (err != vr::EVRSettingsError::VRSettingsError_None) {
```

Comment:
Why are we deleting this?

Reply:
I think it's unnecessary, if we fail then the value is just going to be empty, which can be spotted and fixed in the ui.

Reply:
Fair enough